src/main/java/picard/util/IntervalListTools.java

```diff
@@ -402,15 +416,20 @@ protected int doWork() {
         final IntervalList result = ACTION.act(lists, secondLists);
 
         if (INVERT) {
-            SORT = false; // no need to sort, since return will be sorted by definition.
+            SORT = false; // no need to sort, since uniqued() output will be sorted by definition.
+            UNIQUE = true;
+        }
+
+        if (ACTION == Action.UNION) { // UNION is basically Action.CONCAT with SORT and UNIQUE
+            SORT = true;
```

Comment:
if UNIQUE sorts by default, shouldn't this be `SORT=false`?

Reply:
per the previous implementation, UNION assumes sort and unique are true. so if action is union we have to reproduce that behavior.

Reply:
I guess I'm confused by why the INVERT==true branch claims (in the comments) that theres no need to sort, and here there seems to be a need to sort again.

Reply:
theres a bit of a loop of flags and actions overlapping lol… but I don't know what "UNION" would be with "UNIQUE" false.